gulpfile.babel.js

```diff
@@ -93,10 +98,80 @@ gulp.task('default', ['clean'], cb =>
     'lint',
     'vulcanize',
     'scripts',
+    'generate-service-worker',
     cb
   )
 );
 
+// Copy over the scripts that are used in importScripts as part of the generate-service-worker task.
+gulp.task('copy-sw-scripts', () => {
```

Comment:
I left the task in for now, pending the final location for the runtime caching config (in the Gulp file vs an imported script).  I'll clean it up once the runtime caching is finalized.